src/wallet/wallet.cpp

```diff
@@ -4550,14 +4556,20 @@ void CWallet::LearnAllRelatedScripts(const CPubKey& key)
     LearnRelatedScripts(key, OutputType::P2SH_SEGWIT);
 }
 
-std::vector<OutputGroup> CWallet::GroupOutputs(const std::vector<COutput>& outputs, bool single_coin) const {
+std::vector<OutputGroup> CWallet::GroupOutputs(const std::vector<COutputCoin>& outputs, bool single_coin) const {
     std::vector<OutputGroup> groups;
     std::map<CTxDestination, OutputGroup> gmap;
     CTxDestination dst;
-    for (const auto& output : outputs) {
-        if (output.fSpendable) {
-            CInputCoin input_coin = output.GetInputCoin();
+    for (const auto& out : outputs) {
+        if (out.IsSpendable()) {
+            CInputCoin input_coin = out.GetInputCoin();
 
+            if (input_coin.mwCoin) {
+                groups.emplace_back(input_coin, 1, true, 0, 0);
```

Comment:
Can you add a TODO for us to revisit these values? Right now you're hardcoding depth, ownership, ancestor, & descendant, but we'll want to populate these with the actual values at some point.

Reply:
Just added the TODO on my fork but now see you've merged. Feel free to add the TODO post-hoc.

Reply:
Sorry about that. I didn't realize you hadn't added it yet.